src/mame/mame.lst

```diff
@@ -12561,6 +12561,7 @@ bm1stmix                        // GQ753 (c) 1997 (Japan)
 bm2ndmix                        // GX853 (c) 1998 (Japan)
 bm2ndmxa                        // GX853 (c) 1998 (Japan)
 bm3rdmix                        // GX853 (c) 1998 (Japan)
+bm3rdeaa						// GX853 (c) 1998 (Europe)
```

Comment:
`mame.lst` is supposed to be in alphabetical order so `bm3rdeaa` should come before `bm3rdmix` here. Additionally, you should use spaces instead of tabs here between the game title and comment since everything else is using spaces (it looks different in the raw file view but looks the same on web UI).

Reply:
Fixed.